code/modules/cargo/packsrogue/_rogue.dm

```diff
@@ -1,6 +1,6 @@
 /datum/supply_pack/rogue
 	var/static_cost = FALSE
-	var/randomprice_factor = 0.07
+	var/randomprice_factor = 0.3
```

Comment:
Why increase random price factor for things if the cost is static. Isn't this a bit of a nerf to the merchant?